packages/frontend/src/components/accounts/create/implicit_account/CreateImplicitAccount.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import styled from 'styled-components';
+
+import FormButton from '../../../common/FormButton';
+import Container from '../../../common/styled/Container.css';
+import WhereToBuyNearModal from '../../../common/WhereToBuyNearModal';
+import YourAddress from './YourAddress';
+
+const StyledContainer = styled(Container)`
+    text-align: center;
+
+    .address-title {
+        text-align: left;
+        margin: 40px 0 5px 0;
+    }
+
+    h3 {
+        &.bottom {
+            margin: 70px 0 10px 0;
+        }
+    }
+`;
+
+const StyledBuyButton = styled(FormButton)`
+    &&&& {
+        border-radius: 16px;
+        flex: 1;
+        margin-right: 8px;
+
+        :last-child {
+            margin-right: 0;
+        }
+    }
+`;
+
+const BuyButton = ({
+    amountUSD,
+    onClickBuyButton,
+    moonpayIsAvailable
+}) => (
+    <StyledBuyButton
+        disabled={!moonpayIsAvailable}
+        color='light-blue'
+        onClick={() => onClickBuyButton(amountUSD)}
+    >
+        {amountUSD ? (
+            `$${amountUSD}`
+        ) : (
+            <Translate id='account.createImplicitAccount.customAmount' />
+        )
+        }
```

Comment:
Formatting here is pretty sub-optimal and not scannable.

How about moving this into a helper function? e.g.

```javascript
function getBuyButtonLabel(amountUSD) {
  if(amountUSD) {
    return `$${amountUSD}`
  }

  return ( <Translate id='account.createImplicitAccount.customAmount' /> );
}
```